schemas/did.js

```diff
@@ -8,16 +8,19 @@ const {config} = bedrock;
 require('../lib/config');
 
 const cfg = config['veres-one-validator'];
-
-const pattern = cfg.environment === 'test' ?
-  '^(did\:v1\:test\:nym\:)([-_A-Za-z0-9.]+)$' :
-  '^(did\:v1\:nym\:)([-_A-Za-z0-9.]+)$';
+const prefix = cfg.environment === 'test' ? 'did:v1:test' : 'did:v1';
```

Comment:
I'm going to accept this -- but we're going to have to expand this to accept more than just the `test` prefix in the future, so we can support multiple test nets.